### DISCUSSION TITLE
Unusually long build without any corresponding times action (or log) noted

### DISCUSSION THREAD

**Discussion by sangeet** [gatsbyjs/gatsby#32773] [Help]
There is a long pause before the final step. It's almost as if it just randomly waits and does nothing. How do I figure out what's going on here between these two steps?

Log : https://gist.github.com/sangeet/59e6f7b15addd83da6b23269c3aa7f5e

Note the additional time taken between these two steps which is just not traced anywhere in the logs.

```
success Caching HTML renderer compilation - 8.562s
info Done building in 1219.05839301 sec
```

By long pause I mean, some 15 minutes long
This is the biggest task I was able to find: 

```
success Writing page-data.json files to public directory - 0.198s - 37/37 186.99/s
```

Doesn't explain where the other 800 seconds or so are coming from.
Note, this only happens during Github actions and not locally. Any way I can tap in and figure out what's happening after the "Caching HTML renderer compilation" step causing it to hang for 10+ minutes?

This only happens with builds in Github Actions and I am unable to reproduce this locally.

**Comment by KyleAMathews** [comment]
https://www.gatsbyjs.com/plugins/gatsby-plugin-inspector/?=inspector will generate a CPU trace of your site. You'll need to copy the generated file down to your computer to open in chrome devtools
